### PR TITLE
Jar files are shadowed centrally

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: dataflow/build.gradle [hunk 1]
```diff
@@ -10,9 +10,8 @@ dependencies {
   api "org.plumelib:plume-util:${plumeUtilVersion}"
 
   // External dependencies:
-  // If you add an external dependency, you must shadow its packages both in the dataflow-shaded
-  // artifact (see shadowJar block below) and also in checker.jar (see the comment in
-  // ../build.gradle in the shadowJar block).
+  // If you add an external dependency, you must shadow its packages.
+  // See the comment in ../build.gradle in the shadowJar block.
```
Comment (smillst) [inline dataflow/build.gradle:14]: It has to be shaded in task `createDataflowShaded` below, too.

File: javacutil/build.gradle [hunk 2]
```diff
@@ -15,10 +15,8 @@ dependencies {
   implementation "org.plumelib:reflection-util:${reflectionUtilVersion}"
 
   // External dependencies:
-  // If you add an external dependency, you must shadow its packages both in checker.jar and
-  // and dataflow-shaded.jar.
-  // See the comment in ../build.gradle in the shadowJar block and ../dataflow/build.gradle in
-  // shadowJar block.
+  // If you add an external dependency, you must shadow its packages.
```
Comment (smillst) [inline javacutil/build.gradle:18]: This change is correct.

**Review by smillst — CHANGES_REQUESTED** [review]

**Comment by mernst** [comment]
@smillst Could you go ahead and write this correctly?  Thanks.

**Comment by smillst** [comment]
I've made this change https://github.com/typetools/checker-framework/pull/6074.

**Pull request closed without merging**